parity/cli/mod.rs

```diff
@@ -1350,12 +1350,12 @@ struct Snapshots {
 #[derive(Default, Debug, PartialEq, Deserialize)]
 #[serde(deny_unknown_fields)]
 struct Misc {
-	ntp_servers: Option<Vec<String>>,
 	logging: Option<String>,
 	log_file: Option<String>,
 	color: Option<bool>,
 	ports_shift: Option<u16>,
 	unsafe_expose: Option<bool>,
+	_legacy_ntp_servers: Option<Vec<String>>,
```

Comment:
I know we did this for some other deprecated options, but do we need to keep this fields? Seems like it isn't used anywhere besides when parsing the argument. We can use `or |_| None` in that case.

Reply:
So I removed this field in my last commit. But the other ones are still there. Probably to be removed in the scope of another pr?